src/oldToNewHash.ts

```diff
@@ -0,0 +1,14 @@
+import { hashMap } from "./ipfsHashMap";
+
+/* This method is required due to a bug in uploading Dapp metadata to IPFS.
+ * Old content hashes are retained and cannot be changed due to how contract was written.
+ * This maps old invalid hashes to new valid contents hashes for IPFS.
+ * For more details see: https://github.com/status-im/infra-ipfs/issues/4
+*/
```

Comment:
Don't leave the trailing `*/` on it's own line. There's usually two styles for these kind of multi-line comments:
```
/* something something
 * something something */
```
or
```
/**
 * something something
 * something something
 **/
```

Reply:
Changed it to second style